code/modules/interface/multiContext/context_actions.dm

```diff
@@ -611,15 +603,6 @@
 					playsound(target, 'sound/items/penclick.ogg', 50, 1)
 					return ..()
 
-#undef OMNI_MODE_PRYING
-#undef OMNI_MODE_SNIPPING
-#undef OMNI_MODE_WRENCHING
-#undef OMNI_MODE_SCREWING
-#undef OMNI_MODE_PULSING
-#undef OMNI_MODE_CUTTING
-#undef OMNI_MODE_WELDING
-#undef OMNI_TOOL_WAIT_TIME
```

Comment:
You still still be undefining wait_time no?

Reply:
[a36418c](https://github.com/goonstation/goonstation/pull/12356/commits/a36418ce24fd8cac28c02e94d81bc9d5670f8edb)